packages/enrichPrescriptions/tests/data/simple/statusUpdatesCancelled.json

```diff
@@ -0,0 +1,18 @@
+{
+  "schemaVersion": 1,
+  "isSuccess": true,
+  "prescriptions": [
+    {
+      "onboarded": true,
+      "prescriptionID": "727066-A83008-2EFE36",
+      "items": [
+        {
+          "isTerminalState": "false",
+          "itemId": "e76812cf-c893-42ff-ab02-b19ea1fa11b4",
+          "lastUpdateDateTime": "2023-09-11T13:27:32.000Z",
```

Comment:
I don't think we need more statusUpdate files. The difference between these and the original are very minor and can be made in the code, in a line or two.

Reply:
Unnecessary examples have been removed.